File: src/libs/ThirdParty.VBSuppliers/Properties/Travelgate/Thirdparties/TravelgatePerlatours/TravelgatePerlatours.vb
Imports ThirdParty.Abstractions.Constants
Imports ThirdParty.Abstractions.Lookups

Public Class TravelgatePerlatours
    Inherits Travelgate

    Public Sub New(settings As ITravelgateSettings, support As ITPSupport)
        MyBase.New(settings, support)
    End Sub

    Public Overrides Property Source As String
        Get
            Return ThirdParties.TRAVELGATEPERLATOURS
        End Get
        Set
        End Set
    End Property

End Class

File: src/libs/ThirdParty.CSSuppliers/Travelgate/Thirdparties/TravelgatePerlatours/TravelgatePerlatours.cs
namespace ThirdParty.CSSuppliers.Travelgate
{
    using System.Net.Http;
    using Intuitive.Helpers.Security;
    using Intuitive.Helpers.Serialization;
    using Microsoft.Extensions.Logging;
    using ThirdParty.Constants;
    using ThirdParty.Lookups;

    public class TravelgatePerlatours : Travelgate
    {
        public TravelgatePerlatours(
            ITravelgatePerlatoursSettings settings,
            ITPSupport support,
            HttpClient httpClient,
            ISecretKeeper secretKeeper,
            ISerializer serializer,
            ILogger<TravelgatePerlatours> logger)
            : base(settings, support, httpClient, secretKeeper, serializer, logger)
        {
        }

        public override string Source => ThirdParties.TRAVELGATEPERLATOURS;
    }
}